src/constants.js

```diff
@@ -103,6 +103,7 @@ const requests = {
 const timeouts = {
   BALANCE_PERSIST_FOR: 5 * 60 * 1000,
   CANCEL_ORDER: 5 * 60 * 1000,
+  DEX_SAVE_CHART_STATE: 5000,
   MONITOR_TRANSACTIONS: 10 * 60 * 1000,
```

Comment:
This really could just be a constant local to the chart file, since it will only be used there, but it is ok.